guidelines/component-usage.md

```diff
@@ -1 +1,106 @@
+## Table of content
 
+1. [Syntax](#syntax)
+2. [Formatting](#formatting)
+3. [Component Structure](#component-structure)
+4. [Eslint](#eslint)
+
+
+## Syntax
+
+1. #### Naming Style **[⬆](#table-of-content)**
+
+   * Always use `.jsx` | `.tsx` extensions.
+   * Use UpperCamelCase as a components name.
+   * Use lowerCamelCase for prop names.
+   * It's recommended to use suffixes `Component` or `Container` for explicit definition of "dumb" and "smart" components: `UserProfileComponent` and `UserProfileContainer`.
```

Comment:
I'm not sure what dumb and smart components are.  I would've guessed Component is smart and Container is dumb, but they are listed in the opposite order. :)

Without knowing what a smart component is yet, adding "Component" to a component's name seems redundant to me, personally (and doesn't indicate what makes it smart).